modules/data.remote/inst/gee2pecan_smap.py

```diff
@@ -1,23 +1,22 @@
 """
 Downloads SMAP Global Soil Moisture Data from Google Earth Engine and saves it in a netCDF file.
 
-Data retrieved: ssm, susm, smp, ssma, susma
-
 Requires Python3
 
 Author: Ayush Prasad
 """
-from gee_utils import create_geo, get_sitecoord, get_sitename
+
 import ee
 import pandas as pd
+import geopandas as gpd
 import os
 import xarray as xr
 import datetime
 
 ee.Initialize()
 
 
-def gee2pecan_smap(geofile, outdir, start, end):
+def gee2pecan_smap(geofile, outdir, start, end, var):
```

Comment:
@julbateman unless this is intentional, I think you have the old version of `gee2pecan_smap.py`. In the current version, we discontinued the option for the user to specify their choice of variable and are now downloading all variables/bands/layers by default. We also make use of utility functions. If there isn't any objection can you please make use of the current code? Otherwise, this would break when used via `remote_process()`

https://github.com/PecanProject/pecan/blob/develop/modules/data.remote/inst/gee2pecan_smap.py

Btw so nice to see that this code could be of some use :)

Reply:
Sorry about that! I'll switch it so that it uses the new version. 👍🏼